api/legal.go

```diff
@@ -457,6 +518,13 @@ func (entity *LegalTechnologyManipulating) Valid() (bool, error) {
 	return entity.List.Valid()
 }
 
+// ClearNos clears the "no" answers on application rejection
+func (entity *LegalTechnologyManipulating) ClearNos() error {
+	entity.HasManipulating.ClearNo()
```

Comment:
all of these Legal ones are following the same pattern, too. if we could have that generic type define `ClearNos`, then whatever constructor or initializer would only have to map the generic property type to all these type-specific ones like `HasManipulating`.